types/cache-interceptor.d.ts

```diff
@@ -6,7 +6,7 @@ declare namespace CacheHandler {
   export type CacheMethods = 'GET' | 'HEAD' | 'OPTIONS' | 'TRACE'
 
   export interface CacheOptions {
-    store?: CacheStore
+    store: CacheStore
```

Comment:
```suggestion
    store?: CacheStore
```

Reply:
This was not committed

Reply:
This is different. It's `CacheHandlerOptions` you are thinking of `CacheOptions`

Reply:
Ah!